app/Console/Commands/ResetDemo.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Console\ConfirmableTrait;
+
+class ResetDemo extends Command
+{
+    use ConfirmableTrait;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'demo:reset
+                            {--force : Force the operation to run when in production.}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Reset the demo env';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void
+    {
+        if ($this->confirmToProceed()) {
+            try {
+                Artisan::call('down');
+
+                $this->line('Downlading file...');
+                $file = Http::get('https://github.com/officelifehq/demosql/raw/main/officelife.sql');
+
+                $this->line('Running transaction...');
+                DB::unprepared($file->body());
+
+                $this->line('Running migration...');
```

Comment:
Do we need to run migrations at this stage? The SQL dump file should not have any migrations remaining.

Reply:
I would not be so sured. The sql file is in another repo, and we will forget to upgrade it for time to times!